lib/simple_forms_api_submission/metadata_validator.rb

```diff
@@ -37,12 +37,16 @@ def self.validate_file_number(metadata)
     end
 
     def self.validate_zip_code(metadata, zip_code_is_us_based)
```

Comment:
@Thrillberg With the changes that have been made, I now believe the `zip_code_is_us_based` argument should have a default value of `true`. Otherwise, any use of this module that does not pass a `zip_code_is_us_based` argument is going to result in all zip code values being set to '00000'. Right?

Reply:
I believe it does! https://github.com/department-of-veterans-affairs/vets-api/blob/335b80645a5351fb619f794f49977f78ba159d27/lib/simple_forms_api_submission/metadata_validator.rb#L5

Reply:
Missed that. Apologies.